### ISSUE TITLE
Fix head direction on track position animation

### ISSUE THREAD

**Issue by jsoules** [magland/spikesortingview#66]
I had been reading this data column as the head orientation, in radians, where 0 would consistently mean "parallel to the positive x axis of the measurement coordinate system."

However, @edeno has said that this line should be `position_x` to `poxition_x + r * cos(direction)` and `position_y` to `position_y + r * sin(direction)`.

We're actually drawing a triangle (rather than a line alone) with the head orientation indicated by the tip of the triangle. The code is as follows:

```Typescript
const defaultHeadRadius = 18
const defaultPositionRadius = 10
const defaultTrianglePartialRadius = defaultPositionRadius * 1 // adjusts width of the triangle
const rightAngle = Math.PI/2

    if (frame.headDirection) {
        const localHeadDirection = frame.headDirection
        const headX = frame.x + defaultHeadRadius * cos(localHeadDirection)
        const headY = frame.y + defaultHeadRadius * sin(localHeadDirection)
        const triAX = frame.x + defaultTrianglePartialRadius * cos(localHeadDirection + rightAngle)
        const triAY = frame.y + defaultTrianglePartialRadius * sin(localHeadDirection + rightAngle)
        const triBX = frame.x + defaultTrianglePartialRadius * cos(localHeadDirection - rightAngle)
        const triBY = frame.y + defaultTrianglePartialRadius * sin(localHeadDirection - rightAngle)

        context.beginPath()
        context.moveTo(headX, headY)
        context.lineTo(triAX, triAY)
        context.lineTo(triBX, triBY)
        context.closePath()
        context.fill()
    }
```

So:
- Animal position is (x, y)
- Position is represented by a dot with a radius of 10 pixels.
- We find three points describing the triangle:
  - The tip is at `(x + 18 * cos(direction), y + 18 * sin(direction))`
  - Triangle base point A is at `(x + 10 cos (direction + 90 degrees), y + 10 sin (direction + 90 degrees))`
  - Triangle base point B is at `(x + 10 cos (direction - 90 degrees), y + 10 sin (direction - 90 degrees))`
- Draw the direction triangle by joining up those 3 points (noting that A and B should be on the position dot's circumference)

Am I missing something silly?

Eric, you also mentioned something about polar coordinates? But I think the logic here should be the same...

**Comment by jsoules** [comment]
Eric included a reference video:

https://user-images.githubusercontent.com/2347301/186712724-2c35c1c5-2105-45fb-a46e-b586695f3516.mp4

Compare to https://www.figurl.org/f?v=http://localhost:3000&d=sha1://2fd29200ff6ac8de2a312460f6dbf6a2cc3e4b5a&label=&s={}

**Comment by jsoules** [comment]
So comparison with the video shows some interesting things.

The positions start out 90 degrees off from each other: at the 0 time code, the animal in Eric's video is facing to the upper right, but in my animation it's facing to the bottom right, which suggests I have an off-by-pi/2 error.

However, in seconds 5-6 in Eric's video, the animal is turning clockwise to proceed down the first path. Cf. animation time codes 1581886961 - 181886962, which show the animal turning counterclockwise.

Following up on this, I realized that the head direction display code is happening in pixel space--where the positive-y direction is toward the bottom of the screen, not the top. So we need to flip the sign on the `sin`-based operations that compute the head direction triangle. (Noting that `cos(-x) = cos(x)` and `sin(-x) = -sin(x)`, so we could also just flip the sign of the head orientation, but I would prefer to correct the code.)

Corrected in PR #80.
